src/stories/GenreSelect.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import {ComponentMeta, ComponentStory} from "@storybook/react";
+
+import {useState} from "react";
+
+import GenreSelect from "../components/GenreSelect";
+
+import {GenreDTO} from "../models/GenreDTO";
+
+export default {
+    title: "Components/GenreSelect",
+    component: GenreSelect,
+} as ComponentMeta<typeof GenreSelect>;
+
+const Template: ComponentStory<typeof GenreSelect> = args => {
+    const [selectedGenres, setSelectedGenres] = useState<GenreDTO[]>([]);
+
+    const genres: GenreDTO[] = [
```

Comment:
This constant can be also placed outside the component

Reply:
Done